packages/storybook/stories/QuickStarts/snippets/QuickstartCompositeAdapter.snippet.tsx

```diff
@@ -8,19 +8,32 @@ import {
 } from '@azure/communication-react';
 import React, { useEffect, useMemo, useState } from 'react';
 
+export type CompositeProps = {
```

Comment:
I think we need a ChatCompositeProps and CallCompositeProps, as groupId doesn't apply to Chat and likewise threadId and endpoint doesn't apply to CallComposite

Reply:
Oh I see this is "AllCompositeProps" superset kind of interface, if its only used to pass in props for all composites then leave as is 👍

Reply:
Yeah, the idea was to make a object they could add everything to if they wanted to play with the customization options within the quick start example.